src/sonic-host-services/scripts/caclmgrd

```diff
@@ -109,7 +109,7 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
         self.namespace_docker_mgmt_ip = {}
         self.namespace_docker_mgmt_ipv6 = {}
 
-        namespaces = device_info.get_all_namespaces()
```

Comment:
>device_info.get_all_namespaces [](http://example.com/codeflow?start=21&length=30)

Will you remove the function definition completely?

Reply:
Yes, for all the multi asic specific functions we need to use the ones defined in multi_asic.py. 
The function in device_info.py will be removed